recipes/piano_fingering/meta.yaml

```diff
@@ -0,0 +1,46 @@
+
+{% set name = "piano_fingering" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: piano_fingering
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/p/piano_fingering/piano_fingering-{{ version }}.tar.gz
+  sha256: 737011b979f62742109842c654734167051ea402a545de542e03e915b577070f
+
+build:
```

Comment:
I think this should be `noarch: python`:

https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python